README.md

```diff
@@ -1,3 +1,5 @@
 # discourse-anonymous-moderators
 
 Allows moderators to have a second account for performing actions anonymously. Other staff members can see the real identity of the account.
+
+**Note:** This feature relies on plus addressing for the child account e-mails, and requires `SiteSetting.normalize_emails` to be turned off.
```

Comment:
Might be worth making a site setting validation for `anonymous_moderators_enabled` that prevents it from being turned on if `normalize_emails` is on?

Reply:
💡